packages/jest-types/__typetests__/jest.test.ts

```diff
@@ -574,6 +574,18 @@ expectError(jest.useFakeTimers('modern'));
 expectType<typeof jest>(jest.useRealTimers());
 expectError(jest.useRealTimers(true));
 
+expectType<Promise<number>>(jest.advanceTimersByTimeAsync(250));
```

Comment:
Good to see these (; Could you keep alphabetical order? I mean, `advanceTimersByTimeAsync` should be placed after `advanceTimersByTime`, `advanceTimersToNextTimerAsync` after `advanceTimersToNextTimer` and so on.